packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -2305,25 +2308,40 @@ export const handlers = [
     return HttpResponse.json(response);
   }),
   http.get('*/monitor/services', () => {
-    const response = {
-      data: [{ service_type: 'linode' }],
+    const response: ServiceTypesList = {
```

Comment:
Can we use same logic in UT file as well

Reply:
updated UT